crowbar_framework/app/helpers/application_helper.rb

```diff
@@ -246,11 +246,11 @@ def default_platform
     NodeObject.default_platform
   end
 
-  def available_platforms
-    NodeObject.available_platforms
+  def available_platforms(architecture)
+    NodeObject.available_platforms(architecture)
   end
 
-  def disabled_platforms
-    (NodeObject.disabled_platforms + Crowbar::Repository.disabled_platforms).uniq
+  def disabled_platforms(architecture)
+    (NodeObject.disabled_platforms(architecture) + Crowbar::Repository.disabled_platforms(architecture)).uniq
```

Comment:
Metrics/LineLength: Line is too long. [109/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)